docs/source/examples/workspaces/openstack/topologies/os-network.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Is there a pinfile that references this topology?

Reply:
No. However , This topology is used as a example in documentation.